src/taipy/core/data/data_node.py

```diff
@@ -320,12 +320,12 @@ def write(self, data, job_id: Optional[JobId] = None, **kwargs: Dict[str, Any]):
         from ._data_manager_factory import _DataManagerFactory
 
         self._write(data)
-        self._track_edit(job_id=job_id, **kwargs)
+        self.track_edit(job_id=job_id, **kwargs)
         self.unlock_edit()
         _DataManagerFactory._build_manager()._set(self)
 
-    def _track_edit(self, **options):
-        """Add Edit tracking information to this data node."""
+    def track_edit(self, **options):
+        """Add edit tracking information to this data node."""
```

Comment:
I think we need to explain the `options` parameter as well.

Reply:
yeah... trying to think of a way to explain it actually 🤣

Reply:
we need to say that Taipy interprets some options:
- timestamp
- comments
- job_id
- maybe some others

The others are user custom. Users can use options to attach any information to an external  edit of a dn.